app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/BrowserNavigationOverlay.kt

```diff
@@ -225,19 +225,26 @@ class BrowserNavigationOverlay @JvmOverloads constructor(
     fun showMegaTileError() {
```

Comment:
"This PR includes thorough tests or an explanation of why it does not" is checked, but this PR doesn't include any tests or, as far as I can tell, an explanation of why it doesn't.

Reply:
Oops, #1387 would be the test for this.

Reply:
Once this logic is moved into the VM, it should be testable.  It would definitely be preferable to write tests for this now rather than pushing it off to a new issue, and in general we should try to have at least a few tests in each PR.  Not always possible, but a good goal.  LMK if you need a hand testing this, and I can assist.